Sources/Momento/config/logging/LogProvider.swift

```diff
@@ -1,11 +1,23 @@
+import Logging
```

Comment:
@anitarua I think you are very much on the right track with your choice to use this library. Nice find!

That said, I think the implementation will actually be much simpler than what you are doing now. One of the main goals/benefits of a logging front-end library like the one you found is that the library code (in this case, us) should not have to provide any configuration knobs at all for logging. The user should be able to configure logging entirely independently.

So what I would expect to see is:
1. I think you can delete this class/file entirely
2. I don't think you need to maintain any log level / state in our config objects at all
3. In any of the classes where you want to log something, I think you should just be able to construct a new logger, like this: https://github.com/apple/swift-log/#lets-log
4. our docs / README should mention that we are using this lib, and should link to the example bootstrap line from their docs: https://github.com/apple/swift-log/#lets-log
5. Ultimately (doesn't have to be in this PR) would be nice to provide an example of how to use one of the concrete LogHandler implementations, but we will want to do this in its own project so that the extra dependency only gets included for that one example.

Of the other languages that we have SDKs for, this logging facade you found seems most similar to Java/slf4j. If you poke around our Java SDK code you will see places where we are calling `LoggerFactory.getLogger` from that library, but we don't expose anything related to it in our config because it can be configured independently. I think this will end up feeling similar to that.

Reply:
this one might be worth considering for your example:

https://github.com/vapor/console-kit/blob/main/Sources/ConsoleLoggerExample/entrypoint.swift

Reply:
or https://github.com/sushichop/Puppy